dashboard/static/js/file_access_within_week.js

```diff
@@ -18,6 +18,10 @@ var mini_margin = {top: 50, right: 10, bottom: 50, left: 10},
     mini_width = 100 - mini_margin.left - mini_margin.right,
     mini_height = 400 - mini_margin.top - mini_margin.bottom;
 
+const REMEMBER_MY_SETTING = 'Remember my setting';
```

Comment:
Since this text is duplicated in 2 files I think we should define these in a separate file and include that file before both of these 2 javascript files are included. That would help i18n future consideration.

Reply:
ya. I have been thinking about that. We need to have a solution to have a JS static content loaded from the separate file. I have the issue opened [here](https://github.com/tl-its-umich-edu/my-learning-analytics/issues/361)

Reply:
Thanks, something that would be nice to have for spring/summer.